src/pages/Login.jsx

```diff
@@ -1,12 +1,87 @@
-import React from 'react';
+import { Link } from 'react-router-dom';
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Container from 'react-bootstrap/Container';
 import SearchBar from '../components/SearchBar';
```

Comment:
```suggestion
// import SearchBar from '../components/SearchBar';
```
tá com uns errinhos de lint que vou apontar, tá? queria sugerir todos pra não precisar dar push de novo mas não consigo :/